### PR TITLE
imported the less files

### PR DESCRIPTION
**Pull request by desiquinn**

### CODE REVIEW

**Comment by desiquinn** [comment]
@ayunas I'm done

File: index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (ayunas) [inline index.html:6]: the meta viewport tag is key for getting your media queries to work.  Good job adding it.

File: less/home-page.less [hunk 2]
```diff
@@ -24,6 +29,20 @@
...
+    }
+
+    @media @phone {
+      .sky{
+        display: none;
+      }
+
+      .launch {
+        display: none;
+      }
```
Comment (ayunas) [inline less/home-page.less:45]: This is completely fine to do your media queries the normal css way.  But when you're using less, you can actually use nesting to put media queries nested inside your elements. that way you don't have to declare the element inside of the query.

File: less/home-page.less [hunk 3]
```diff
@@ -58,7 +77,25 @@
       }
 
       .btn {
+        button{
+          .button(@btn-bg-color, @porcelain);
+
+          &:hover {
+            .buttonHover(@porcelain, @btn-bg-color)
```
Comment (ayunas) [inline less/home-page.less:84]: nice job using the parametric mixin.

File: less/home-page.less [hunk 4]
```diff
@@ -58,7 +77,25 @@
...
+            width: 100%;
+            height: 60px;
+            padding: 20px;
+          }
+        }
+      }
+
+      @media @phone {
+        width: 100%;
       }
```
Comment (ayunas) [inline less/home-page.less:97]: which element is this media query for?

File: less/home-page.less [hunk 5]
```diff
@@ -70,6 +107,17 @@
...
+      @media @phone {
+        width: 100%;
+        margin: 20px 0;
+      }
+    }
+
+    @media @phone {
+      flex-direction: column-reverse;
+      margin-top: 20px;
+      padding: 15px;
```
Comment (ayunas) [inline less/home-page.less:120]: Also hard to tell what element this media query is for.  you might want to do the indenting for your less so it's easier to read.  Or use a VScode extension like Prettier to do it for you.
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

File: less/navigation.less [hunk 6]
```diff
@@ -1 +1,45 @@
...
+
+    .flexinColumn();
+
+    h1 {
+        margin-bottom: 15px;
+        animation-name: space;
+        animation-duration: 10s;
+        animation-delay: 2s;
+        animation-iteration-count: 3;
+    }
```
Comment (ayunas) [inline less/navigation.less:28]: I really love your animation of the SpaceWalkers header.  Really well done! @desiquinn

File: less/variables.less [hunk 7]
```diff
@@ -24,3 +25,4 @@
 @border-real: #dee2e6;
 
 // Mobile var
+@phone: ~"(max-width: 500px)";
```
Comment (ayunas) [inline less/variables.less:28]: great job doing the escape string for the mobile variable.

File: css/index.css [hunk 8]
```diff
@@ -0,0 +1,336 @@
...
+.home .walk .walk-text .btn button {
+  border: 1px solid #22283A;
+  background-color: #22283A;
+  border-radius: 10px;
+  color: #F6F7F8;
+  width: 50%;
+  height: 40px;
+  padding: 10px 0;
+  font-size: 1.7rem;
+  font-weight: bold;
```
Comment (ayunas) [inline css/index.css:296]: add a transition : .5s here, and when the user moves the mouse off the button, it fades back to the other color.

File: css/index.css [hunk 9]
```diff
@@ -0,0 +1,336 @@
...
+}
+.home .walk .walk-text .btn button:hover {
+  border: 1px solid #F6F7F8;
+  background-color: #F6F7F8;
+  color: #22283A;
```
Comment (ayunas) [inline css/index.css:301]: as we discussed, add a transition : .5s here to give your button a nice fading color effect.

**Review by ayunas — APPROVED** [review]
Great looking project @desiquinn.  you're doing amazing.  keep up the good work.

**Pull request merged**